ocaml/xapi-cli-server/cli_frontend.ml

```diff
@@ -1045,6 +1045,15 @@ let rec cmdtable_data : (string * cmd_spec) list =
       ; flags= [Host_selectors]
       }
     )
+  ; ( "host-rescan-drivers"
+    , {
+        reqd= ["uuid"]
+      ; optn= []
+      ; help= "Scan the host and update its driver information."
+      ; implementation= No_fd Cli_operations.host_rescan_drivers
+      ; flags= []
```

Comment:
You can use `Host_selectors` here rather than the `uuid` arg above, since this is on the host. It's a bit more flexible for users.